packages/mocktail/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ repository: https://github.com/felangel/mocktail
 homepage: https://github.com/felangel/mocktail/tree/main/packages/mocktail
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.15.0 <3.0.0"
 
```

Comment:
I’d recommend only bumping the sdk if we need to take advantage of new features otherwise we’re just restricting the dart sdk versions unnecessarily and projects that are still using dart 2.12.0 will no longer be able to use mocktail.

Reply:
I don't think it's needed because `any()` will infer the type automatically.